cmd/nse/nse.go

```diff
@@ -161,3 +260,18 @@ func socketCleanup(listenEndpoint string) error {
 	}
 	return nil
 }
+
+func getCurrentNS() string {
+	buf := make([]byte, MaxSymLink)
+	numBytes, err := syscall.Readlink(netnsfile, buf)
```

Comment:
Why syscall.Readlink over os.Readlink?

Reply:
os.Readlink returns an error and *not* the content of the link if the link is broken.

/proc/self/ns/net is a link, but its *always* a broken link.  The netns inode is in the link content... os.Readlink doesn't work.

Reply:
right